tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3678,6 +3678,24 @@ def extract_glimpse(
 
   Returns:
     A `Tensor` of type `float32`.
+    
+  Usage Example:
+    ```python
+    >> import tensorflow as tf
```

Comment:
please remove >> symbol

Reply:
@rthadur , Hey I removed the >> symbol. Thanks

Reply:
thank you